### PR TITLE
added licenses and repos for Beacon Network

### PR DESCRIPTION
**Pull request by costero-e**
Adding the repositories for the Beacon Network product (backend and UI).
Can you please review and accept @brunopacheco1 ?
Thank you,
Oriol

### CODE REVIEW

**Comment by brunopacheco1** [comment]
@costero-e thanks a lot for your PR.

Can you please also add these two repo in `.github/workflows/report_per_repo.yml`?

Then you will be able to get ORT repo for each of them.

**Comment by costero-e** [comment]
hi @brunopacheco1,

I have added them now.

Tell me if now it's good to do the merge, please.

Thank you,

Oriol

File: .github/workflows/report_per_repo.yml [hunk 1]
```diff
@@ -46,8 +46,10 @@ jobs:
             sw-name: starter-kit-lsaai
           - repository: neicnordic/sensitive-data-archive
             sw-name: neicnordic/sensitive-data-archive
-          - repository: NBISweden/sda-cli
```
Comment (brunopacheco1) [inline .github/workflows/report_per_repo.yml:49]: You accidentally removed the repo `NBISweden/sda-cli`. Can you please add it back?

**Review by brunopacheco1 — CHANGES_REQUESTED** [review]

**Comment by costero-e** [comment]
Hi @brunopacheco1, I'm sorry, I added it back now.

I hope everything is ready now to be merged.

Best,

Oriol

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**